website/public/index.html

```diff
@@ -235,13 +235,13 @@ <h2>
                     in one place, and government teams can better reach
                     communities in need.
                   </p>
-                  <a href="/about" class="btn-action-primary m-t-1"
+                  <a href="/learnmore" class="btn-action-primary m-t-1"
                     ><span class="btn-action-title">Learn more</span
                     ><span class="btn-action-text"
                       >Read about how CiviForm works</span
                     ></a
                   >
-                  <a href="/contact" class="btn-action-primary m-t-1"
+                  <a href="/getstarted-new" class="btn-action-primary m-t-1"
```

Comment:
**suggestion:** getstarted-new would be what users see when they navigate there, I think - do we want that?

Reply:
Yes. Some cleanup is required on the file and directory structure since there is a bug in the website generator software. https://github.com/11ty/eleventy/discussions/2294